home/.chezmoiscripts/universal/run_onchange_before_11-install-wazuh-agent.sh.tmpl

```diff
@@ -0,0 +1,141 @@
+    elif command -v dnf || command -v yum > /dev/null; then
+        # RHEL based OS
+        logg info 'Setting up repository to install Wazuh Agent'
+        sudo rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH
+        sudo tee -a /opt/wazuh.repo << EOF
+[wazuh]
+gpgcheck=1
+gpgkey=https://packages.wazuh.com/key/GPG-KEY-WAZUH
+enabled=1
+name=EL-$releasever - Wazuh
+baseurl=https://packages.wazuh.com/4.x/yum/
+protect=1
+EOF
+        logg info 'Installing and registering Wazuh Agent'
+        WAZUH_MANAGER="{{ .host.wazuh.manager }}" sudo yum install -y wazuh-agent
+    elif command -v apk > /dev/null; then
+        # Alpine
+        logg info 'Setting up repository to install Wazuh Agent'
+        wget -O /etc/apk/keys/alpine-devel@wazuh.com-633d7457.rsa.pub https://packages.wazuh.com/key/alpine-devel%40wazuh.com-633d7457.rsa.pub
+        echo "https://packages.wazuh.com/4.x/alpine/v3.12/main" | sudo tee -a /etc/apk/repositories
+        sudo apk update
+        logg info 'Installing and Wazuh Agent'
+        sudo apk add wazuh-agent
+        logg info 'Registering Wazuh Agent'
+        export WAZUH_MANAGER="{{ .host.wazuh.manager }}" && sudo sed -i "s|MANAGER_IP|$WAZUH_MANAGER|g" /var/ossec/etc/ossec.conf
+    elif command -v pacman > /dev/null; then
+        # Archlinux
+        logg info 'Installing build dependencies'
+        sudo pacman --noconfirm -Syu curl gcc make sudo wget expect gnupg perl-base perl fakeroot python brotli automake autoconf libtool gawk libsigsegv nodejs base-devel inetutils cmake jq
+        logg info 'Finding the latest version of Wazuh Agent'
+        WAZUH_AGENT_VERSION=$(curl -s https://api.github.com/repos/wazuh/wazuh/releases/latest | jq -r '.tag_name')
+        logg info 'Downloading the latest version of Wazuh Agent'
+        cd /tmp
+        curl -Ls "https://github.com/wazuh/wazuh/archive/$WAZUH_AGENT_VERSION.tar.gz" | tar zx
+        logg info 'Installing the latest version of Wazuh Agent'
+        cd wazuh*
+        tee -a etc/preloaded-vars.conf << EOF
+USER_LANGUAGE="en"
+USER_NO_STOP="y"
+USER_INSTALL_TYPE="agent"
+USER_DIR="/var/ossec"
+USER_ENABLE_SYSCHECK="y"
+USER_ENABLE_ROOTCHECK="y"
+USER_ENABLE_OPENSCAP="y"
+USER_ENABLE_ACTIVE_RESPONSE="y"
+USER_AGENT_SERVER_IP="{{ .host.wazuh.manager }}"
+USER_CA_STORE="n"
+EOF
+        ./install.sh
+    elif command -v zypper > /dev/null; then
+        # OpenSUSE
+        logg info 'Setting up repository to install Wazuh Agent'
+        sudo rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH
+        sudo tee /etc/zypp/repos.d/wazuh.repo << EOF
+[wazuh]
+gpgcheck=1
+gpgkey=https://packages.wazuh.com/key/GPG-KEY-WAZUH
+enabled=1
+name=EL-$releasever - Wazuh
+baseurl=https://packages.wazuh.com/4.x/yum/
+protect=1
+EOF
+        sudo zypper refresh
+        logg info 'Installing and registering Wazuh Agent'
+        WAZUH_MANAGER="{{ .host.wazuh.manager }}" sudo zypper install wazuh-agent
+    fi
+
+    {{- if eq .host.distro.family "linux" -}}
```

Comment:
Can you use `if [ -d /Applications ] && [ -d /System ]` for macOS detection?

Reply:
Done.